server/agoraApi.yaml

```
@@ -180,7 +180,490 @@ paths:
+      requestBody:
+        description: Include a Resource object to save in JSON format *NOTE* - Only the resouce member is required, id will be ignored.
+        required: true
+        content: 
+          application/json:
+            schema: 
+              type: object
+              $ref: '#/components/schemas/Resouce'
+      responses:
+        '201':
+          description: New Resource created
+          content:
+            application/json:
+              schema:
+                type: object
+                $ref: '#/components/schemas/Resource'
+          headers:
+            x-agora-message-title:
+              schema:
+                type: string
+              description: Title of Agora API message
+            x-agora-message-message:
+              schema:
+                type: string
+              description: Message of Agora API message
+        '200':
+          description: Updated existing record - the resource name already existed, current record used.
+            application/json:
+              schema:
+                type: object
+                $ref: '#/components/schemas/Resource'
+          headers:
+            x-agora-message-title:
+              schema:
+                type: string
+              description: Title of Agora API message
+            x-agora-message-message:
+              schema:
+                type: string
+              description: Message of Agora API message
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+        '403':
+          $ref: '#/components/responses/ForbiddenError'
+  /resources/user/{id}:
+    get:
+      tags: 
+        - Resources
+      summary: Retrives of all active resources available to a specified user
+      parameters:
+        - in: path
+          name: id
+          schema:
+            type: integer
+          description: id of the user to retrive their active resources
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                $ref: '#/components/schemas/Resource'
+        headers:
+            x-agora-message-title:
+              schema:
+                type: string
+              description: Title of Agora API message
+            x-agora-message-message:
+              schema:
+                type: string
+              description: Message of Agora API message
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/UnauthorizedError'
+        '403':
+          $ref: '#/components/responses/ForbiddenError'
+        '404':
+          $ref: '#/components/responses/NotFoundError'
+  /resources/shared:
+    get:
+      tags:
+        - Resources
+      summary: Retrives of all shared resources available
```

Spelling / additional conciseness

```suggestion
      summary: Retrieves all available shared resources
```